### PR TITLE
fix(DB/Creature) Highperch Soarer

### PR DESCRIPTION
**Pull request by nolt**
## Changes Proposed:
-  fix non targatable two mobs Highperch Soarer

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/10227

## SOURCE:

https://www.wowhead.com/npc=6139/highperch-soarer

## Tests Performed:

- updates to DB
- check mobs are they targetable, killable, lootable

## How to Test the Changes:

1. .go c 21708, .go c 21706
2. target, kill, loot mob

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by nolt** [comment]
I propuse to make them killable, in retail they were flying above trees with eggs and was unreachable to anyone.

**Comment by Nyeriah** [comment]
Hey,

Thanks for your PR, it's much appreciated. However, I have a question: you said it yourself they were not reachable and could not be killed on retail. What exactly motivated you to change that?

I remember there were cosmetic gryphons flying ontop of the trees that could not be killed too, I don't think we should go out of our way to interfere with that unless we know for sure it's wrong currently on AC.

**Comment by nolt** [comment]
I will check today if I place them higher than before they will not fall to the ground. One of them had spawn point in the middle of sky but he fall on the ground while the other one moving path was totally broken.

**Comment by Nyeriah** [comment]
Yes, but should they fall to the ground at all? Shouldn’t they just fly in the background? (Scenery?)

If they were never meant to be killed, no reason to change it imo

**Comment by acidmanifesto** [comment]
> I will check today if I place them higher than before they will not fall to the ground. One of them had spawn point in the middle of sky but he fall on the ground while the other one moving path was totally broken.

try checking their creature id in the creature template movement table to see if they have set to a 2 for fly for can fly?

**Comment by nolt** [comment]
@Nyeriah 
No no, one of them just flow into textures, another one fall on the ground.
@acidmanifesto in creature_template movementType was 0, but in creature one of them has 2.

I changed their spawn as close as it possible to retail.
Zrzut ekranu z 2022-01-20 21-38-55
Zrzut ekranu z 2022-01-20 21-39-30

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1642639045919873206.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1642639045919873206');
+
+#closes #10227
+UPDATE creature SET `position_x`=-5026.72, `position_y`=-949.15, `position_z`=61.9471, `orientation`=4.07623 WHERE `guid`=21706;
+UPDATE creature SET `position_x`=-5195.02, `position_y`=-1213.47, `position_z`=116.165, `orientation`=1.10743, `MovementType`=0 WHERE `guid`=21708;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1642639045919873206.sql:5]:
```suggestion
UPDATE `creature` SET `position_x`=-5026.72, `position_y`=-949.15, `position_z`=61.9471, `orientation`=4.07623 WHERE `guid`=21706;
UPDATE `creature` SET `position_x`=-5195.02, `position_y`=-1213.47, `position_z`=116.165, `orientation`=1.10743, `MovementType`=0 WHERE `guid`=21708;
```

**Review by Kitzunu — DISMISSED** [review]

**Pull request merged**